ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -3210,6 +3210,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-registry-nutanix-object-p2-f28
+  cluster: build01
+  cron: 15 4 10 * *
```

Comment:
pls add `remote_api: true` for the job.

Reply:
@jianlinliu  Will add, thanks. But what's use for?

Reply:
so that users can run curl command via remote api to trigger the job.